admin/tabs/wizard.php

```diff
@@ -152,5 +169,129 @@ public function load_options( $options, $step ) {
           </form>
           <?php
     }
+    public function load_plugins( $plugins, $step ) {
+        ?>
+    <form onsubmit="installPlugin(event)">
+    <table class="widefat striped">
+          <thead>
+            <tr>
+              <th colspan="3"><?php echo 'Set Plugins' ?></th>
+            </tr>
+          </thead>
+          <tbody>
+          <?php
+            foreach ( $plugins as $plugin ){
+                $slug = $plugin['slug'];
+                ?>
+            <tr>
+              <td>
+                <?php echo esc_html( $slug ); ?>
+              </td>
+                <?php
+                if ( is_plugin_installed( $slug ) ){
+                  //if ( is_plugin_active( $slug . '/' . $slug . '.php' )){
+                    if ( is_plugin_activated( $slug ) ){
+                        ?>
+                      <td>
+                        <?php echo 'Active' ?>
+                      </td>
+                        <?php
+                    } else {
+                        ?>
+                      <td>
+                        <input type="hidden" name="<?php echo esc_html( json_encode( $plugin ) )  ?>" value="<?php echo esc_html( wp_create_nonce( 'updates' ) ) ?>" />
```

Comment:
You only need this nonce to be in the form once. It should be on the line immediately following the <form> tag and have a `name` of `ajax_nonce` or something like that.